spec/dummy/db/schema.rb

```diff
@@ -172,6 +175,7 @@
     t.text "choices"
     t.text "settings"
     t.text "custom_expectations"
+    t.integer "educational_purpose", default: 0
```

Comment:
The term educational seems redundant to me, since mostly everything in laboratory is education-oriented

Reply:
I agree with that but he have a _type_ and a _goal_ field in exercise. Adding just purpose seems too vague in this context.

Reply:
I disagree.  Type and goal have clear semantics.  The first is the exercise style like problem or playground. The latter is a term extensively used in mumukit, domain and runners and describes a way of evaluation - as opposed to tests and expectations.

Educational does not remove any ambiguity here.

Reply:
The name comes from a discussion we had a couple months ago. I commented many other options we had considered on the [original commit](https://github.com/mumuki/mumuki-domain/commit/0280fff33cb3035b3dc66bf03321b843527eec1e) on Domain, maybe there's something there that suits the idea better.

Reply:
If we want to remove any ambiguity perhaps we should use the term flow or didactic sequence.

Or something like motivation. motivation_in_flow for instance.